installing/installing_vsphere/installing-vsphere.adoc

```diff
@@ -0,0 +1,47 @@
+[id="installing-vsphere"]
+= Installing a cluster on vSphere
+include::modules/common-attributes.adoc[]
+:context: installing-vsphere
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
```

Comment:
Most clusters won't be a single host. I would reword this.

Reply:
What would I need to change to support multiple hosts? @staebler said that you had to do some things differently, but I don't have the info. (I do have an issue to add the information.)

Reply:
@dav1x Are you recommended that we just do not mention using a single host here? There install docs are not describing how a customer would install onto multiple hosts, but such an installation is possible.

Something like the following?
```
you can install a cluster on vSphere infrastructure that you provision.
```

Reply:
@staebler That looks great. A usual source for provisioning would be a cluster (vmware level not OCP).

Reply:
@dav1x, @staebler, can you tell me more about what a user would need to do to install on multiple hosts?

Reply:
@kalexand-rh , nothing special would need to be done. The UPI assigns VMs to a resource pool, resource pools are tied to vsphere clusters (or single hosts).